### PR TITLE
Auto heal rule

### PR DESCRIPTION
**Pull request by craigmaher**
Adding an auto heal rule to prod slots

### CODE REVIEW

File: infrastructure/ArmTemplates/antareskvtemplate.json [hunk 1]
```diff
@@ -352,7 +352,29 @@
...
+                      "subStatus": 0,
+                      "win32Status": 0,
+                      "count": 100,
+                      "timeInterval": "00:10:00"
+                  }
+                ],
+                "slowRequests": null
+              },
+              "actions": {
+                "actionType": "Recycle",
```
Comment (ShekharGupta1988) [inline infrastructure/ArmTemplates/antareskvtemplate.json]: can we collect dumps and then recycle

Comment (ShekharGupta1988) [reply infrastructure/ArmTemplates/antareskvtemplate.json]: Also is there a way to recycle for StatusCode >=500

Comment (craigmaher) [reply infrastructure/ArmTemplates/antareskvtemplate.json]: there is no way to specify status code above 500 we can specify multiple unique status codes though. are there any others you can think of?

Comment (ShekharGupta1988) [reply infrastructure/ArmTemplates/antareskvtemplate.json]: 503 would be good to add.
The substatus and win32 is optional I believe as per Autoheal UI, so we can prob ignore those

**Pull request merged**